### PR TITLE
fix: removed all references to direct chat and group chat

### PR DESCRIPTION
**Pull request by disha1202**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #

**Did you add tests for your changes?**

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

## Release Notes

- **New Features**
	- Enhanced error handling for admin creation and membership-related operations.
	- Improved event management capabilities with a new recurrence rule for events.
	- Added a new field for user permissions regarding plugin management.

- **Bug Fixes**
	- Corrected field requirements in the chat schema.

- **Refactor**
	- Streamlined GraphQL schema by removing outdated input types related to messaging and chat functionalities.
	- Updated terminology across the codebase from "DirectChat" to "Chat," enhancing clarity and consistency.

- **Tests**
	- Updated test descriptions and structures to reflect changes in terminology from "DirectChat" to "Chat."

### CODE REVIEW

**Comment by palisadoes** [comment]
Please make adjustments to the code so that code rabbit approves the PR

**Comment by palisadoes** [comment]
Disha, please make sure coderabbit approves the changes

File: tests/resolvers/Chat/creator.spec.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ afterAll(async () => {
   await disconnect(MONGOOSE_INSTANCE);
 });
 
-describe("resolvers -> DirectChat -> creator", () => {
+describe("resolvers -> Chat -> creator", () => {
```
[the remark this answers is not in the document]
Comment (disha1202) [reply tests/resolvers/Chat/creator.spec.ts:23]: Removed all the references

File: src/resolvers/Query/chatsByUserId.ts [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { Chat } from "../../models";
  * This query will fetch all the Chats for the current user from the database.
  * @param _parent-
  * @param args - An object that contains `id` of the user.
- * @returns An object `directChats` that contains all direct chats of the current user.
- * If the `directChats` object is null then it throws `NotFoundError` error.
+ * @returns An object `chats` that contains all chats of the current user.
+ * If the `Chats` object is null then it throws `NotFoundError` error.
```
[the remark this answers is not in the document]
Comment (disha1202) [reply src/resolvers/Query/chatsByUserId.ts]: Fixed and updated the documentation

**Comment by disha1202** [comment]
> Disha, please make sure coderabbit approves the changes

Done

**Pull request merged**